common/perception_utils/include/perception_utils/object_classification.hpp

```diff
@@ -0,0 +1,41 @@
+
+// Copyright 2022 TIER IV, Inc.
```

Comment:
[style] An unnecessary empty line? :thinking: 

```suggestion
// Copyright 2022 TIER IV, Inc.
```